write command to download csv file

terrazoon opened the pull request:
## Description

The developer can get the s3 file location for a problematic csv file from the admin logs. Write a command so the developer can download the csv file contents securely.


## Security Considerations

N/A

xlorepdarkhelm commented on app/commands.py:
```diff
@@ -597,7 +597,10 @@ def download_csv_file_by_name(csv_filename):
+    content = s3.get_s3_file(bucket_name, csv_filename, access_key, secret, region)
+    # We want it all to get logged in one line
+    content = content.replace("\n", "|")
+    print(content)
```
Is `print()` better than using a logger with logging levels?

terrazoon replied:
If we use logger, the phone numbers will be sanitized.  We actually want to get the phone numbers (and emails) here, because we think something is wrong with the file.

xlorepdarkhelm reviewed (approved):
LGTM

ccostino reviewed (changes requested):
Thanks @terrazoon for attempting to make the debugging easier here for folks!

Unfortunately, we can't move forward with this as is, because by exposing the phone numbers/emails in the logs, we run the risk of them being exposed in addition to violating the very principles on safeguarding this data that we've outlined in our SSPP and reviewed with our assessors to meet a variety of NIST controls about PII.

Since this is within the command that would download the file anyway, what were you hoping to achieve or help with by having the contents of the file itself put in the logs, other than the tedium?

terrazoon commented:
> Thanks @terrazoon for attempting to make the debugging easier here for folks!
> 
> Unfortunately, we can't move forward with this as is, because by exposing the phone numbers/emails in the logs, we run the risk of them being exposed in addition to violating the very principles on safeguarding this data that we've outlined in our SSPP and reviewed with our assessors to meet a variety of NIST controls about PII.
> 
> Since this is within the command that would download the file anyway, what were you hoping to achieve or help with by having the contents of the file itself put in the logs, other than the tedium?

yeah, brain problems.  I changed it.

terrazoon commented:
> > Thanks @terrazoon for attempting to make the debugging easier here for folks!
> > Unfortunately, we can't move forward with this as is, because by exposing the phone numbers/emails in the logs, we run the risk of them being exposed in addition to violating the very principles on safeguarding this data that we've outlined in our SSPP and reviewed with our assessors to meet a variety of NIST controls about PII.
> > Since this is within the command that would download the file anyway, what were you hoping to achieve or help with by having the contents of the file itself put in the logs, other than the tedium?
> 
> yeah, brain problems. I changed it.

@ccostino I changed it again.  Now using the s3.download_file() functionality from both 3.  I had to unship a random test also to keep coverage at 95%

ccostino reviewed (approved):
Great, thanks @terrazoon!  This looks like it's ready to go now; we ought to be able to test it in staging to be sure!
